### PR TITLE
Update msi-deployment.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5496
Added to important note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/msi-deployment.md [hunk 1]
```diff
@@ -60,7 +60,12 @@ The Teams MSI will place an installer in Program Files. Whenever a user signs in
 Do not use the MSI to deploy updates, because the client will auto update when it detects a new version is available from the service. To re-deploy the latest installer use the process of redeploying MSI described below. If you deploy an older version of the MSI package, the client will auto-update (except in VDI environments) when possible for the user. If a very old version gets deployed, the MSI will trigger an app update before the user is able to use Teams.
 
 > [!Important]
```
Comment (JohanFreelancer9) [inline Teams/msi-deployment.md:62]:
```suggestion
> [!IMPORTANT]
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Reached out to PM.

**Pull request merged**